src/components/cc-logs-instances/cc-logs-instances.stories.js

```diff
@@ -0,0 +1,324 @@
+import './cc-logs-instances.js';
+import { makeStory } from '../../stories/lib/make-story.js';
+import { enhanceStoriesNames } from '../../stories/lib/story-names.js';
+
+/**
+ * @typedef {import('./cc-logs-instances.types.d.ts').Instance} Instance
+ * @typedef {import('./cc-logs-instances.types.d.ts').Deployment} Deployment
+ */
+
+function randomCommit () {
+  return randomString(40, 'abcdef0123456789');
+}
+function randomString (length = 8, alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz') {
```

Comment:
nitpick: could you break a line between the two functions?
question: I've had to use a similar function and I could use it for `cc-product-card` instead of what I did, could we put it in a lib file too?

Reply:
I'll do that